samples/scanner/scanners/firewall_rules/blacklist.yaml

```diff
@@ -0,0 +1,154 @@
+    #     "name": "firewall2",
+    #     "network": "https://www.googleapis.com/compute/v1/projects/yourproject/global/networks/default",
+    #     "priority": 1000,
+    #     "sourceRanges": ["0.0.0.0/0"],
+    #     "allowed": [
+    #       {
+    #         "IPProtocol": "all",
+    #       }
+    #     ],
+    #     "direction": "EGRESS",
+    #   }
+  - rule_id: 'prevent_allow_all'
+    description: Prevent allow all policies
+    type: blacklist
+    match_policies:
+      - direction: ingress
+        allowed: ['*']
+    verify_policies:
+      - allowed:
+        - IPProtocol: 'all'
+
+    # This rule will trigger on policies that expose all ports for either TCP or UDP.
+    #
+    # Example triggering firewall rule:
+    #   {
+    #     "name": "firewall1",
+    #     "network": "https://www.googleapis.com/compute/v1/projects/yourproject/global/networks/default",
+    #     "priority": 1000,
+    #     "sourceRanges": ["10.0.0.0/8"],
+    #     "allowed": [
+    #       {
+    #         "IPProtocol": "tcp",
+    #         "ports": ["all"]
+    #       }
+    #     ],
+    #     "direction": "INGRESS",
+    #   }
+    #
+    # Example firewall rule that WON'T trigger:
+    #   {
+    #     "name": "firewall2",
+    #     "network": "https://www.googleapis.com/compute/v1/projects/yourproject/global/networks/default",
+    #     "priority": 1000,
+    #     "sourceRanges": ["10.0.0.0/8"],
+    #     "denied": [
+    #       {
+    #         "IPProtocol": "tcp",
+    #         "ports": ["all"]
+    #       }
+    #     ],
+    #     "direction": "INGRESS",
+    #   }
+  - rule_id: 'disallow_any_port'
+    description: Don't allow policies that expose every port
+    type: blacklist
+    match_policies:
+      - direction: ingress
+        allowed: ['*']
+    verify_policies:
+      - allowed:
+        - IPProtocol: 'tcp'
+          ports:
+            - 'all'
+      - allowed:
+        - IPProtocol: 'udp'
+          ports:
+            - 'all'
+
+    # This rule will trigger on policies that allow IPProtocol all.
+    #
+    # Example triggering firewall rule:
+    #  {
+    #    "name": "firewall1",
+    #    "network": "https://www.googleapis.com/compute/v1/projects/yourproject/global/networks/default",
+    #    "priority": 1000,
+    #    "sourceRanges": ["0.0.0.0/0"],
+    #    "targetTags": ["linux"],
+    #    "allowed": [
+    #      {
+    #        "IPProtocol": "tcp",
+    #        "ports": ["3389"]
+    #      }
+    #    ],
+    #  }
+    #
+    # Example firewall rule that WON'T trigger:
+    #
+    #  {
+    #    "name": "firewall2",
+    #    "network": "https://www.googleapis.com/compute/v1/projects/yourproject/global/networks/default",
+    #    "priority": 1000,
+    #    "sourceRanges": ["0.0.0.0/0"],
+    #    "targetTags": ["notlinux"],
+    #    "allowed": [
+    #      {
+    #        "IPProtocol": "tcp",
+    #        "ports": ["3389"]
+    #      }
+    #    ],
+    #  }
+  - rule_id: 'no_rdp_to_linux'
```

Comment:
This example rule does not match the comments right above:
```# This rule will trigger on policies that allow IPProtocol all.```

I do like this example, so perhaps add the right comments for this rule, and then add the right rule example to the above comment?